src/GhostGeneration.jl

```diff
@@ -0,0 +1,128 @@
+# defines and stores relevant variables to ghost creation and area calculation
+struct GhostedArea
+    resolution::Int
+    n_ghosts::Int
+    rapidity_step::Float64
+    phi_step::Float64
+    ghost_pt::Float64
+
+    # constructor
+    # default value of ghost_pt set to 1.0e-45 as -45 is the smallest magnitude that doesn't result in the value defaulting to 0
+    function GhostedArea(resolution::Int)
+        new(resolution,
+            resolution * resolution,
+            Float64(10.0 / resolution),
```

Comment:
Same comment as below about the `10.0` -- this should be determined by a user-specified rapidity range.